scripts/arm32_ci_script.sh

```diff
@@ -212,7 +212,7 @@ do
         ;;
     --softfp)
```

Comment:
I found that we can modify `-softfp` in `corefx/netci.groovy` script.
I will update the last patch again soon.